.github/PULL_REQUEST_TEMPLATE/new-pep.md

```diff
@@ -0,0 +1,72 @@
+---
+name: New PEP
+about: Submit a new draft PEP for consideration
+labels: "new-pep"
+---
+
+<!--
+
+Please read and follow PEP 1 and PEP 12 before submitting a new PEP:
+
+https://peps.python.org/pep-0001/#submitting-a-pep
+
+https://peps.python.org/pep-0012/#how-to-use-this-template
+
+Make sure to include the PEP number in the pull request title; for example:
+
+PEP 9999: <Title of PEP>
+
+See our Contributing Guidelines (CONTRIBUTING.rst) for more information.
+
+Thanks!
+
+-->
+
+
+# New PEP
+
+<!--
+
+Briefly describe your new PEP here, and anything else we should know.
+Be sure to link to any previous discussions or issues motivating/drafting it.
+
+-->
+
+
+
+## Checklist
+
+<!--
+
+You should use the following as a guide when double-checking your PEP,
+and you can help complete some of it yourself if you like
+by ticking any boxes you're sure about, like this: [x]
+
+If you're unsure about anything, just leave it blank and we'll take a look.
+
+-->
+
```

Comment:
```suggestion

<!--

Resist the temptation to describe your PEP here.
Your PEP's Abstract and Post-History should speak for itself.

You can use the following checklist when double-checking your PEP,
and you can help complete some of it yourself if you like
by ticking any boxes you're sure about, like this: [x]
If you're unsure about anything, just leave it blank and we'll take a look.
-->

```

Reply:
```suggestion

<!--

You can use the following checklist when double-checking your PEP,
and you can help complete some of it yourself if you like
by ticking any boxes you're sure about, like this: [x]
If you're unsure about anything, just leave it blank and we'll take a look.

-->

```

The above, minus the extra "resist the temptation" text—this hasn't ever really been an actual problem in the past, and the more structured nature of this template should likely further dissuade if it is. Additionally, a brief meta-description with some additional contextually-relevant details doesn't exactly _hurt_, if authors feel the need to provide it.